src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -286,3 +286,18 @@ void Terminal::ColorSelection(const COORD, const COORD, const TextAttribute)
 {
     THROW_HR(E_NOTIMPL);
 }
+

```

Comment:
nit: I'd probably just move this to `Terminal.cpp`, I think this file's supposed to be specific to our selection handling